packages/augur-sdk/src/utils.ts

```diff
@@ -124,3 +127,37 @@ export function calculatePayoutNumeratorsValue(
 
   return payout.findIndex((item: string) => parseInt(item, 10) > 0).toString();
 }
+
+export function calculatePayoutNumeratorsArray(
+  displayMaxPrice,
+  displayMinPrice,
+  numTicks,
+  numOutcomes,
+  marketType,
+  outcome
+): BigNumber [] {
+  const payoutNumerators = Array(numOutcomes).fill(new BigNumber(0));
+  const isScalar = marketType === MarketTypeName.Scalar;
+
+  if (isScalar) {
+    // selectedOutcome must be a BN as string
```

Comment:
`selectedOutcome` doesn't seem to be a thing

Reply:
I'll fix that